Dockerfile

```diff
@@ -95,6 +100,25 @@ RUN sip-install \
 WORKDIR /output/
 RUN find /upstream/ -name \*.pyi -exec cp {} . \;
 
+################################################################################
+# PyQt5-SIP stubs
+################################################################################
+
+FROM build-dep AS sip
+
+# Reuse arguments from previous build scope
+ARG SIP_VERSION
+ARG SIP_ABI_VERSION
+
+# Install SIP including stubs
+# TODO: Find way to build only stubs. This takes way too long
```

Comment:
This TODO comment is probably not relevant as we're not building anything here.

Reply:
```suggestion
```
Agreed.